_layouts/post.html

```diff
@@ -132,4 +131,4 @@ <h3 class="font-weight-bold">Summary</h3>
     </div>
 {% endif %}
 <!--End Comments
-================================================== -->
+================================================== -->
```

Comment:
```suggestion
================================================== -->

```

끝부분은 언제나 newline 넣어주세요!